packages/app/src-tauri/src/commands/ftp.rs

```diff
@@ -31,7 +31,13 @@ pub async fn ftp_upload(payload: Payload) -> Result<String, String> {
         return Err("Failed to login to server: {}".to_string());
     }
 
-    ftp_stream.set_mode(Mode::ExtendedPassive);
+    // https://www.iana.org/assignments/ftp-commands-extensions/ftp-commands-extensions.xhtml
+    if ftp_stream
+        .feat()
+        .is_ok_and(|opts| opts.get("EPSV").is_some())
+    {
+        ftp_stream.set_mode(Mode::ExtendedPassive);
```

Comment:
@newfla what do you think if we enable it by a configuration flag. 
We cannot assume that the connection will always be a passive connection?
I mean to keep this check but also verify if a passiveMode plag is true in the payload.

Reply:
As default we can set to Mode::Passive, then check for the EPSV flag. Set_mode simply stores the flag so should not be a performance/protocol issue. We can ignore active mode